code-challenges.js

```diff
@@ -1,63 +1,106 @@
-// ASSESSMENT 2: Coding Practical Questions
+// April Rieger - ASSESSMENT 2: Coding Practical Questions
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 // Use the test variables provided.
 
 var num1 = 15
+
+const myFunction = () => {
+    if(num1 % 3 === 0){
+        return `${num1} is divisible by three`
+    } else {
+        return `${num1} is not divisible by three`
+    }
+}
```

Comment:
how can we do this with just a single function? hint: passing the numbers in as an argument

Reply:
Hi Rachael,
On review of our assessment last Monday morning I realized that the prompt was different than I had thought. So if I were to pass all three numbers into a single function I would do it like this: 

const myFunction = (number) => {
    if(number % 3 ===0){
        return `${number} is divisible by 3.`
    } else if(number% 3 !== 0){
        return `${number} is not divisible by 3.`
    } else {
        return "error"
    }
}
console.log(myFunction(num1))
console.log(myFunction(num2))
console.log(myFunction(num3))